Scripts/Commands/ConvertPlayers.cs

```diff
@@ -51,9 +51,9 @@ public static void Convert_OnCommand(CommandEventArgs e)
                 NetState.ProcessDisposedQueue();
                 World.Save();
             
-                Console.WriteLine("{0} players have been converted to PlayerMobile. {1}.", count, Core.Service ? "The server is now restarting" : "Press any key to restart the server");
+                Console.WriteLine("{0} players have been converted to PlayerMobile. {1}.", count, Core.NoConsole ? "The server is now restarting" : "Press any key to restart the server");
                 
-                if (!Core.Service)
+                if (!Core.NoConsole && !Core.Service)
```

Comment:
No. Service and NoConsole have two different intended purposes. Please see my other replies.